tools/make/docs.mk

```diff
@@ -101,8 +101,9 @@ docs-release-prepare:
 .PHONY: docs-release-gen
 docs-release-gen:
 	@$(LOG_TARGET)
-	@$(call log, "Added Release Doc: site/content/en/$(TAG)")
-	cp -r site/content/en/latest site/content/en/$(TAG)
+	$(eval DOC_VERSION := $(shell cat VERSION | cut -d "." -f 1,2))
+	@$(call log, "Added Release Doc: site/content/en/$(DOC_VERSION)")
+	cp -r site/content/en/latest/ site/content/en/$(DOC_VERSION)/
 	@for DOC in $(shell ls site/content/en/latest/user); do \
```

Comment:
this had been broken for a long time, https://github.com/envoyproxy/gateway/pull/3766 will help.